src/Map.jsx

```diff
@@ -130,6 +131,35 @@ class Map extends Component {
   /**
    * Calculate & Update state of new dimensions
    */
+
+  setNewCity(city) {
+    const cities = [
+      {
+        name: 'San Francisco',
+        latitude: 37.7853,
+        longitude: -122.41669,
+      },
+      {
+        name: 'Toronto',
+        latitude: 43.761539,
+        longitude: -79.411079,
+      },
+    ];
+
+    let selectedCity = {};
+
+    for (let i = 0; i < cities.length; i += 1) {
+      if (cities[i].name === city.value) selectedCity = cities[i];
+    }
```

Comment:
You could use some cooler syntax here
`const selectedCity = cities.find(city => city.name === newCity.value);`

Reply:
Thanks for your feedback! I submitted a new commit with the mentioned changes.